apps/admin-gui/src/app/vos/components/application-form-list/application-form-list.component.ts

```diff
@@ -52,11 +52,60 @@ export class ApplicationFormListComponent implements OnChanges {
   mapForCombobox: Map<number, string> = new Map();
   dragDisabled = true;
 
+  // labels for hidden and disabled icons
+  ifEmpty = this.translate.instant('VO_DETAIL.SETTINGS.APPLICATION_FORM.DISABLED_HIDDEN_ICON.IF_EMPTY');
+  ifPrefilled = this.translate.instant('VO_DETAIL.SETTINGS.APPLICATION_FORM.DISABLED_HIDDEN_ICON.IF_PREFILLED');
+
+  // tooltips for hidden and disabled icons
+  alwaysDisabled = this.translate.instant('VO_DETAIL.SETTINGS.APPLICATION_FORM.DISABLED_HIDDEN_ICON.ALWAYS_DISABLED_HINT');
+  alwaysHidden = this.translate.instant('VO_DETAIL.SETTINGS.APPLICATION_FORM.DISABLED_HIDDEN_ICON.ALWAYS_HIDDEN_HINT')
+  isDisabledIf = this.translate.instant('VO_DETAIL.SETTINGS.APPLICATION_FORM.DISABLED_HIDDEN_ICON.DISABLED_IF_HINT');
+  isHiddenIf = this.translate.instant('VO_DETAIL.SETTINGS.APPLICATION_FORM.DISABLED_HIDDEN_ICON.HIDDEN_IF_HINT');
+  isEmpty = this.translate.instant('VO_DETAIL.SETTINGS.APPLICATION_FORM.DISABLED_HIDDEN_ICON.IS_EMPTY_HINT');
+  isPrefilled = this.translate.instant('VO_DETAIL.SETTINGS.APPLICATION_FORM.DISABLED_HIDDEN_ICON.IS_PREFILLED_HINT');
```

Comment:
Such initialization should be performed in the `onInit` method. It might be also a cause, why some of the translations are not properly loaded. 

![image](https://user-images.githubusercontent.com/26108901/119803082-6a93fd00-bedf-11eb-80d4-d8739c388bcd.png)

Reply:
Initialization moved to `ngOnInit()` method.